src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs

```diff
@@ -775,11 +812,93 @@ private CodeGenOutput GenerateFastifyPluginOutput(ServiceInfo service)
 					}
 				}
 
+				WriteJsonSchemaDtos(code, service);
+
 				if (TypeScript)
 					WriteTypes(code, httpServiceInfo);
 			});
 
 			return new CodeGenOutput(file);
+
+			string GetJsonSchemaType(ServiceTypeInfo? serviceType)
+			{
+				if (serviceType is null)
+					throw new ArgumentNullException(nameof(serviceType));
```

Comment:
`ServiceInfo.GetServiceType` can return `null`, so I opted to handle that explicitly here instead using `!` at the call sites. What is the scenario where this will be null?

Reply:
I think it would only be null if the type name was invalid, which shouldn't be possible if the service parsed successfully.

Reply:
Will change to use `!` at the call sites then. Thanks!

Reply:
fixed: 51f5c1f2d35349c5037b5613cb3740cd934dc7a4